packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -12,13 +20,63 @@ import { Panel } from '@fluentui/react';
 export const VideoEffectsPane = (props: {
   showVideoEffectsOptions: boolean;
   setshowVideoEffectsOptions: (showVideoEffectsOptions: boolean) => void;
+  adapter: CallAdapter | CommonCallAdapter;
 }): JSX.Element => {
-  return VideoEffectsPaneTrampoline(props.showVideoEffectsOptions, props.setshowVideoEffectsOptions);
+  const { showVideoEffectsOptions, setshowVideoEffectsOptions } = props;
+  /* @conditional-compile-remove(video-background-effects) */
```

Comment:
Is this an internal component that will only be used for the video-background-effects feature? If yes, I think we don't need the conditional compilation statements.

Reply:
But you are already not showing the VideoEffectsPane in stable by having a CC statement in CallWithChatComposite.tsx
![image](https://user-images.githubusercontent.com/79475487/228715575-d84f7e86-95ea-44df-a866-b6100cbc3f04.png)

Reply:
I'll still approve to unblock :) but I still think the CC lines are unnecessary

Reply:
Thanks Miguel, for pointing this out. I will merge this PR to get unblocked and have created a separate item to remove the CC from video effects pane https://skype.visualstudio.com/SPOOL/_workitems/edit/3207057